specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/examples/StorageAccountCreate.json

```diff
@@ -12,6 +12,8 @@
       "location": "eastus",
       "properties": {
         "isHnsEnabled": true,
+        "allowBlobPublicAccess": false,
```

Comment:
As update account also support these 2 properties, please also add sample for update account.

Reply:
As per offline talk. SRP supports PUT as PATCH and examples are pretty much the same. We do not advertise all the properties in update requests. I would like to keep it this way as the properties are sensitive and I want to prevent accidental toggle of these values in case the user directly pics and run the examples.

Reply:
Since this sample will be used in validate Swagger, and in Rest API doc, we should make the sample aligned with the swagger. 
And as we will also open the update in SDK/Tools, only hide it in swagger sample don't make much sense.

Reply:
I have added the properties in the update example